Managers/UInputMan.cpp

```diff
@@ -980,6 +980,14 @@ namespace RTE {
 			if (m_DisableMouseMoving && m_PrepareToEnableMouseMoving && (mousePosX >= 0 && mousePosX < g_FrameMan.GetResX() && mousePosY >= 0 && mousePosY < g_FrameMan.GetResY())) {
 				m_DisableMouseMoving = m_PrepareToEnableMouseMoving = false;
 			}
+		} else if (g_InActivity == false) {
```

Comment:
Seems like this could be rewritten a little more cleanly with no duplicated code.
Extract the mousewheel handling stuff outside of the first if and put it in another if (mousePlayer != Players::NoPlayer || g_InActivity == false) { //handle mouse wheel...